plutus-core/plutus-core/src/PlutusCore/Default/Builtins.hs

```diff
@@ -178,13 +178,14 @@ data DefaultFun
     | Ripemd_160
     -- Batch 6
     | ExpModInteger
-    | CaseList
-    | CaseData
     | DropList
     -- Arrays
     | LengthOfArray
     | ListToArray
     | IndexArray
+    -- Case
+    | CaseList
```

Comment:
Does the order of constructors here matter?

Reply:
> Does the order of constructors here matter?

No, I don't think so.  The main thing that should never change after deployment is the flat tag.  I think there are some things (tests etc) that iterate over the enumeration of all the builtins, so rearranging the constructors may alter the order that things happen in.  I just moved these to the end to make them easy to find.